_delphi_utils_python/tests/validator/test_static.py

```diff
@@ -237,6 +237,20 @@ def test_empty_df(self):
         validator.check_bad_geo_id_value(empty_df, FILENAME, "county", report)
         assert len(report.raised_errors) == 0
 
+    def test_state_level_fips(self):
+        validator = StaticValidator(self.params)
+        report = ValidationReport([])
+        df = pd.DataFrame(["37183", "56000", "04000", "60000", "78000"], columns=["geo_id"])
+        validator.check_bad_geo_id_value(df, FILENAME, "county", report)
+
+        assert len(report.raised_errors) == 0
+
+        df = pd.DataFrame(["37183", "56000", "04000", "79000" "60000", "78000"], columns=["geo_id"])
```

Comment:
Is there supposed to be a comma between "79000" and "60000"? or is this intentionally a typo. If the former, perhaps do something like 

```suggestion
        df = pd.DataFrame(["37183", "56000", "04000", "60000", "78000", "99000"], columns=["geo_id"])
```
so the first 5 are the same as the valid values in the example above, and the last one is the new and more obviously invalid value